### PR TITLE
Fix typo - trailing > in introductory paragraph

### PR DESCRIPTION
**Pull request by hamishwillee**
Trivial typo fix - removing `>` at end of first sentence

### CODE REVIEW

File: files/en-us/web/api/rtcpeerconnection/addtrack/index.md [hunk 1]
```diff
@@ -7,13 +7,10 @@ browser-compat: api.RTCPeerConnection.addTrack
...
-The {{domxref("RTCPeerConnection")}} method
-**`addTrack()`** adds a new media track to the set of tracks
-which will be transmitted to the other peer.>
+The {{domxref("RTCPeerConnection")}} method *`addTrack()`** adds a new media track to the set of tracks which will be transmitted to the other peer.
 
-> **Note:** Adding a track to a connection triggers renegotiation by
-> firing a {{DOMxRef("RTCPeerConnection/negotiationneeded_event", "negotiationneeded")}} event. See
-> [Starting negotiation](/en-US/docs/Web/API/WebRTC_API/Signaling_and_video_calling#starting_negotiation) for details.
+> **Note:** Adding a track to a connection triggers renegotiation by firing a {{DOMxRef("RTCPeerConnection/negotiationneeded_event", "negotiationneeded")}} event.
+>  See [Starting negotiation](/en-US/docs/Web/API/WebRTC_API/Signaling_and_video_calling#starting_negotiation) for details.
```
Comment (dawei-wang) [inline files/en-us/web/api/rtcpeerconnection/addtrack/index.md:13]:
```suggestion
> See [Starting negotiation](/en-US/docs/Web/API/WebRTC_API/Signaling_and_video_calling#starting_negotiation) for details.
```

File: files/en-us/web/api/rtcpeerconnection/addtrack/index.md [hunk 2]
```diff
@@ -7,13 +7,10 @@ browser-compat: api.RTCPeerConnection.addTrack
 
 {{APIRef("WebRTC")}}
 
-The {{domxref("RTCPeerConnection")}} method
-**`addTrack()`** adds a new media track to the set of tracks
-which will be transmitted to the other peer.>
+The {{domxref("RTCPeerConnection")}} method *`addTrack()`** adds a new media track to the set of tracks which will be transmitted to the other peer.
```
Comment (dawei-wang) [inline files/en-us/web/api/rtcpeerconnection/addtrack/index.md:10]:
```suggestion
The {{domxref("RTCPeerConnection")}} method `_addTrack()_` adds a new media track to the set of tracks which will be transmitted to the other peer.
```

**Review by dawei-wang — COMMENTED** [review]
linting

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**